src/app.js

```diff
@@ -4,7 +4,7 @@ import ReactDOM from 'react-dom';
 import Main from "./components/Main";
 import {H5PContext} from './context/H5PContext';
 
-H5PEditor.widgets.threeImage = H5PEditor.ThreeImage = (function () {
+H5PEditor.widgets.threeImage = H5PEditor.NDLAThreeImage = (function () {
```

Comment:
Do we need to namespace the `H5PEditor.widgets.threeImage` as well? 

```suggestion
H5PEditor.widgets.NDLAthreeImage = H5PEditor.NDLAThreeImage = (function () {
```